packages/web-react/src/components/Pagination/demo/index.tsx

```diff
@@ -0,0 +1,201 @@
+// Because there is no `dist` directory during the CI run
+/* eslint-disable import/no-extraneous-dependencies, import/extensions, import/no-unresolved */
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment, import/extensions, import/no-unresolved
+// @ts-ignore: No declaration file
+import icons from '@lmc-eu/spirit-icons/dist/icons';
+import { IconsProvider } from '../../../context';
+import DocsSection from '../../../../docs/DocsSections';
+import PaginationCurrentFirst from '../stories/PaginationCurrentFirst';
+import PaginationCurrentMiddle from '../stories/PaginationCurrentMiddle';
+import PaginationCurrentLast from '../stories/PaginationCurrentLast';
+import PaginationCurrentFirstCentered from '../stories/PaginationCurrentFirstCentered';
+import UncontrolledPaginationLogicExample from '../stories/UncontrolledPaginationLogicExample';
+import PaginationItem from '../PaginationItem';
+import PaginationLink from '../PaginationLink';
+import PaginationLinkNext from '../PaginationLinkNext';
+import PaginationLinkPrevious from '../PaginationLinkPrevious';
+import UncontrolledPagination from '../UncontrolledPagination';
+
+ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
+  <React.StrictMode>
+    <IconsProvider value={icons}>
+      <DocsSection title="Pagination Current First">
+        <PaginationCurrentFirst>
```

Comment:
Yes, unfortunately it is necessary to list individual items there, because the `Story.args` do not get into the demo and therefore it has to be redefined here.

@literat If you want, we can talk about it, if you don't solve this differently

Reply:
Can we do this the same way as, for example, the Alert demo? We need argTypes only for the first story, which BTW should be named the same way as the component.
<img width="271" alt="Screenshot 2023-06-12 at 12 55 59" src="https://github.com/lmc-eu/spirit-design-system/assets/2481010/428e8c2c-4a4b-4188-b7d8-a435aef3a5b2">

All other "demos" can be imported from stories, I think.

Reply:
But there is no "default" for Pagination. There is neither a Web nor a Twig package

Reply:
So if we said that the examples should be 1:1, that's why I set it up exactly the same as it was for Web and Twig

Reply:
So they are the same except that I added the live demo in React